xds/third_party/xds/import.sh

```diff
@@ -41,10 +41,14 @@ xds/annotations/v3/versioning.proto
 xds/core/v3/authority.proto
 xds/core/v3/collection_entry.proto
 xds/core/v3/context_params.proto
+xds/core/v3/extension.proto
```

Comment:
`xds/third_party/envoy/import.sh` also imports an `extension.proto` but I can see that they are different although `TypedExtensionConfig` definition is identical so looks like this one is ultimately meant to replace the other one. Just pointing out and most probably not a problem and something we are forced to live with ...

Reply:
Interesting find, thank you. I checked in the envoy repo; they don't use `TypedExtensionConfig` from cncf/xds `extension.proto` yet. I don't think there's anything we can do until envoy does the migration.
Regarding matchers - AFAIK they're slightly different, unfortunately. I believe there will be unification, and the migration to cncf/xds.